### ISSUE TITLE
no security at path

### ISSUE THREAD

**Issue by bymaximus** [Mermade/openapi-gui#19]
theres no security related data at paths operations?

**Comment by MikeRalphson** [comment]
Correct, there is no security at [PathItem](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#pathItemObject) level. Though I suspect you're asking about at [Operation](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#operationObject) level, and yes, this is still on the [specification support TODO list](https://github.com/Mermade/openapi-gui/blob/master/docs/openapi3-support.md#operation-object).

If you're asking about the old (red and blue) v2 for Swagger/OpenAPI 2.0 only, then this is also a [TODO](https://github.com/Mermade/openapi-gui/blob/openapi2/README.md#todo) item, but this branch is no-longer maintained, unless someone volunteers to take it on.

**Comment by MikeRalphson** [comment]
Hi @bymaximus the ability to set the security at operation level to `default`, `none` or `custom` and to select the set of custom `securitySchemes` is done. Not yet done is editing scopes for `oauth2` schemes.

If you toggle the `custom security` checkbox it will bring in all the defined security schemes and you can delete the ones you don't want.
